packages/grafana-ui/src/components/Table/DefaultCell.tsx

```diff
@@ -10,7 +10,7 @@ import { getTextColorForBackground, getCellLinks } from '../../utils';
 export const DefaultCell: FC<TableCellProps> = (props) => {
   const { field, cell, tableStyles, row, cellProps } = props;
 
-  const displayValue = field.display!(cell.value);
+  const displayValue = field.display!(cell.value, cell.row.index);
```

Comment:
:tada:

Reply:
nice that this works --- and would finally give us a good path to have color table cells, but I am pretty nervous about the parallel value+index given closures etc

Reply:
> nice that this works --- and would finally give us a good path to have color table cells,

what do you mean by that? I mean we have had colored cells since the start, in what new way?  (beyond the obvious color from another field but the use case for that seems small)